cleanlab/internal/outlier.py

```diff
@@ -33,18 +35,17 @@ def transform_distances_to_scores(distances: np.ndarray, k: int, t: int) -> np.n
 
     Parameters
     ----------
-    distances : np.ndarray
-        An array of distances of shape ``(N, num_neighbors)``, where N is the number of examples.
-        Each row contains the distances to each example's `num_neighbors` nearest neighbors.
-        It is assumed that each row is sorted in ascending order.
-
-    k : int
-        Number of neighbors used to compute the average distance to each example.
-        This assumes that the second dimension of distances is k or greater, but it
-        uses slicing to avoid indexing errors.
+    avg_distances : np.ndarray
+        An array of distances of shape ``(N)``, where N is the number of examples.
+        Each row represents an example's average distance to its k nearest neighbors.
```

Comment:
Shouldn't it be "each example represents .." since array is only a single dimension (N,)

Reply:
'entry' not 'example'